agents/sbd/fence_sbd.py

```diff
@@ -404,6 +430,32 @@ def sbd_daemon_is_running():
 
     return True
 
+def unquote(s):
+    """
+    Reverse shell-quoting a string, so the string '"a b c"'
+    becomes 'a b c'
+    """
+    sp = shlex.split(s)
+    return sp[0] if sp else ""
+
+def parse_sysconfig(sysconfig_file):
```

Comment:
systemd ```EnvironmentFile=``` doesn't have the file executed by a shell but does parsing rather.
So a lot of stuff that would require a shell like variable-references aren't working anyway.
But things like special meaning of a ```\``` are implemented in systemd. Anyway did you check
if we need to do that here at all or if we can use the environment inherited from pacemaker so
that we wouldn't have to deal with checking for compatibility in parsing with systemd. Sry didn't
have the chance to check myself since my last comments.

Reply:
Removed `parse_sysconfig`, reading environment value `SBD_OPTS` instead